swap/src/cli/command.rs

```diff
@@ -90,3 +111,11 @@ fn parse_btc(str: &str) -> Result<bitcoin::Amount> {
     let amount = bitcoin::Amount::from_str_in(str, ::bitcoin::Denomination::Bitcoin)?;
     Ok(amount)
 }
+
+fn parse_multiaddr(str: &str) -> Multiaddr {
```

Comment:
Are you certain these are necessary? I think structopt uses `FromStr` by default so you should be able to get away with not passing a parsing function at all.

Haven't tried it though :)

Reply:
Event better that what I suggested below :D

Reply:
Yep you are right